.eleventy.js

```diff
@@ -112,7 +112,7 @@ const processImage = async imgElem => {
 
   imgElem.setAttribute('loading', 'lazy');
   imgElem.setAttribute('data-src', imgElem.src);
-  imgElem.classList.add(className);
+  imgElem.classList.add(...className);
```

Comment:
If we replaced this line with the following it would enable backwards compatibility for people who are using the older string config setting:
```suggestion

  const classNameArr = Array.isArray(className) ? className : [className];
  imgElem.classList.add(...classNameArr);
```